sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -1414,3 +1414,66 @@ ALTER TABLE `issue_encounter` ADD COLUMN `created_at` DATETIME DEFAULT CURRENT_T
 #IfMissingColumn issue_encounter updated_at
 ALTER TABLE `issue_encounter` ADD COLUMN `updated_at` DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'timestamp when this issue encounter record was last updated';
```

Comment:
hi @adunsulag ,
Can't figure out what happened on this diff? Was there any formatting changes. Unable to manually review this in this form.